robotd/master.py

```diff
@@ -11,12 +11,23 @@
 import pyudev
 import setproctitle
 
-from robotd.devices import BOARDS
+import textwrap
+
+from robotd.devices import BOARDS, STARTUP_BOARDS
+
+
+def _send(connection, message):
+    # Split the message into buf_size chunks
+    buf_size = BoardRunner.SOCK_BUFFER_SIZE
+    for msg in [message[i:i + buf_size] for i in range(0, len(message), buf_size)]:
+        connection.send(msg)
```

Comment:
I disagree with your original statement, it's up to one message per read() call, see the Socket API Spec text about SOCK_SEQPACKET:

> A record can be sent using one or more output operations and received using one or more input operations, but a single operation never transfers parts of more than one record.